A72-open-telemetry-tracing.md

```diff
@@ -105,94 +133,181 @@ class OpenTelemetryPluginBuilder {
+  "Delayed LB pick complete" upon creation of the stream on the transport. 
+* When the application sends an outbound message to the transport, add an Event 
+  with name "Outbound message sent" and the following attributes:
+  * key `message.event.type` with string value "SENT".
+  * key `message.message.id` with integer value of the seq no. The seq no. is a sequence
+    of integer numbers starting from 0 to identify sent messages within the stream, the same below.
+  * key `message.event.size.uncompressed` with integer value of uncompressed 
+    message size. The size is the total attempt message bytes without encryption,
+    not including grpc or transport framing bytes, the same below.
+  * If any compression, key `message.event.size.compressed` with integer value
+    of compressed message size.
+* When an inbound message has been received from the transport, add an Event
+  with name "Inbound message read" and the following attributes:
+  * key `message.event.type` with String value "RECEIVED".
+  * key `message.message.id` with integer value of the seq no.
+  * key `message.event.size.compressed` with integer value of wire message size. 
+  * If any compression, key `message.event.size.uncompressed` with integer value
+    of uncompressed message size.
 * When the stream is closed, set RPC status and end the attempt span.
 
 At the server:
-* When an outbound message has been sent, add message events to capture seq no.,
-  type(SENT) and uncompressed message size, and compressed message size if any compression.
-* When an inbound message has been read from the transport, add message events
-  to capture seq no., type(Received), wire message size, and uncompressed message size,
-  if any decompression.
+* When the application sends an outbound message to the transport, add an Event
+  with name "Outbound message sent" and the following attributes:
+  * key `message.event.type` with string value "SENT".
+  * key `message.message.id` with integer value of the seq no.
+  * key `message.event.size.uncompressed` with integer value of uncompressed
+    message size.
+  * If any compression, key `message.event.size.compressed` with integer value
+    of compressed message size.
+* When an inbound message has been read from the transport, add an Event
+  with name "Inbound message read" and the following attributes:
+  * key `message.event.type` with string value "RECEIVED".
+  * key `message.message.id` with integer value of the seq no.
+  * key `message.event.size.compressed` with integer value of wire message size.
+  * If any compression, key `message.event.size.uncompressed` with integer value
+    of uncompressed message size.
 * When the stream is closed, set the RPC status and end the span.
 
-Note that C++ is missing the seq no. information. And Java has issue of reporting
-decompressed message size upon receiving messages,
+### Limitations
+Note that C++ is missing the seq no. information due to lack of transport support.
```

Comment:
Where does this limitation come from?  I don't see why we would have trouble reporting the sequence numbers.  @yashykt

Reply:
Yeah, I can add it, but I just don't see the point of it. Why do we want to keep track of the sequence number of a message? We don't reorder messages.